cranelift/codegen/src/machinst/valueregs.rs

```diff
@@ -4,6 +4,12 @@
 use regalloc::{RealReg, Reg, VirtualReg, Writable};
 use std::fmt::Debug;
 
+#[cfg(feature = "arm32")]
```

Comment:
Would it be preferable here to key it on the word size of the target that the resulting CL will be compiling for?  Is that even possible, given the limitations of the Rust config etc system?

Reply:
Such an option does exist, but the wrench in the works is cross-compilation -- e.g. `clif-utils` is normally built with all targets enabled (so we need to support arm32 compilation even on an x64 host, etc).

Reply:
(Ah, sorry, actually I misread your comment -- pointer size of target, not host; I don't think we have a config option for that as we have custom features for each backend.)